examples/docs_beta_snippets/docs_beta_snippets/guides/tutorials/etl_tutorial/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
this shouldnt be necessary since its included in the .gitignore of the root folder